inbm-lib/inbm_common_lib/utility.py

```diff
@@ -13,7 +13,7 @@
 
 from dataclasses import dataclass
 from pathlib import Path
-from typing import Iterable, List, Optional, Union
+from typing import Iterable, Optional, Union
```

Comment:
Consider using `from __future__ import annotations` to enable postponed evaluation of type annotations, which can help with forward references and readability. [medium]